GliaWidgets/Component/Button/Action/ActionButtonStyle.swift

```diff
@@ -18,16 +18,16 @@ public struct ActionButtonStyle: Equatable {
     public var backgroundColor: ColorType
 
     /// Corner radius of the button.
-    public var cornerRaidus: CGFloat?
+    public var cornerRaidus: CGFloat
```

Comment:
Seems like this a change of **public** API, this is likely to affect integrators.
@dukhovnyi @gersonnoboa do you think that this a breaking change?

Reply:
Yes, it seems like a breaking change, you cannot send `nil` anymore. It doesn't seem that necessary to remove the `?`, or am I missing something? Otherwise it's too much effort to deprecate this. Another issue is that this says `cornerRaidus` and not `cornerRadius`...

Reply:
I see your reasoning and that's a good point. My thought process was that if it's optional, but we assign a value through init() regardless, there would be no reason for optional. But I understand now. reverted back [7f7ffb0](https://github.com/salemove/ios-sdk-widgets/pull/459/commits/7f7ffb066e97d3911b01d19d5d1c775aa675a8ff)

Reply:
CornerRaidus -> CornerRadius [c8e833e](https://github.com/salemove/ios-sdk-widgets/pull/459/commits/c8e833e8afdbdc9d37d56da2e33f5685ff45a714)